src/services/mock/checkpoint/get_checkpoint.py

```diff
@@ -0,0 +1,32 @@
+query = '''
+{
+    workflows {
+        id
+        name
+        status
+        owner
+        host
+        port
+        taskProxies {
+            id
+            state
+            cyclePoint
+            task {
+                name
+            }
+            jobs {
+                id
+                state
+                submitNum
+            }
+        }
+    }
+}
+'''
```

Comment:
We could generate this from queries registered on the views to save having to make changes in two places but thats a reasonably involved piece of work. Can raise as follow-on if others think it's sensible.

Reply:
It would be better to keep the Python source somewhere else. Either a new repository, or in a directory that is easy to be excluded when producing the binary to be deployed in environments.

Reply:
I though we were only going to distribute the `dist` dir so this wouldn't matter. Otherwise I could migrate the logic to JS?